public/video-ui/src/components/Video/Display.js

```diff
@@ -28,6 +28,7 @@ class VideoDisplay extends React.Component {
   saveAndUpdateVideo = video => {
     if (this.props.route.mode === 'create') {
       this.props.videoActions.createVideo(video);
+      this.props.videoActions.getUsages(this.props.video.id);
```

Comment:
Can we do something to ensure that the video is created before usages are fetched? If the video takes a long time to create is it still possible here to fetch the usages before we have an id?

Reply:
Good call! Have updated to do this once `createVideo` promise has finished